core/src/test/resources/sql/cast-with-format.iq

```diff
@@ -1155,7 +1108,7 @@ EXPR$0
 !ok
 
 !if (false) {
-### disabled until Bug.CALCITE_6269_FIXED ###
+### unsupported format model/elements, test disabled ###
```

Comment:
this seems the same as the test in the change above on line 1091 above which is enabled

Reply:
Agreed I think I missed that one, good catch!